### PR TITLE
Tdrmapper for smallRNAseq pipelines

### PR DESCRIPTION
**Pull request by lpantano**
Adding a tool to quantify and visualize tRNA fragments from small RNA data.

Let me know if the `build.sh` makes sense or it is better to change something.

### CODE REVIEW

File: recipes/tdrmapper/build.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+target=$PREFIX/tdrmapper
```
Comment (johanneskoester) [inline recipes/tdrmapper/build.sh]: A more standard place would be ``$PREFIX/lib/tdrmapper``.

File: recipes/tdrmapper/build.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+mkdir $target
+mkdir -p $PREFIX/bin
+cp Scripts/*pl Scripts/*r $target/.
```
Comment (johanneskoester) [inline recipes/tdrmapper/build.sh]: The ``/.`` is not needed.

File: recipes/tdrmapper/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+source:
+    git_url: https://github.com/lpantano/tDRmapper
+    git_rev: 06e0b9e
+
+build:
+  number: 1
+
+requirements:
+  run:
+
```
Comment (johanneskoester) [inline recipes/tdrmapper/meta.yaml]: Would be good to have a test case here. E.g. try out ``TdrMappingScripts.py --help`` or something like that.

**Comment by lpantano** [comment]
thanks! will work on that changes!

**Comment by lpantano** [comment]
@johanneskoester is good to go now? i see i need to update with master but something else need to do to merge?
thanks!

**Pull request merged**